index.html

```diff
@@ -43,7 +43,7 @@ <h1 >I’m Esmatullah <span><br></span> Glad to see you!</h1>
         <section id="works">
             <div class="projects">
                 <div>
-                    <img class="project-img project-left-img" src="images/first project.svg" alt="">
```

Comment:
:white_check_mark: Image text alternatives** (img alt="alt text") (We added appropiate alt texts to every image)